tests/JsonQueryWrapper/JsonQueryTest.php

```diff
@@ -73,4 +73,43 @@ public function testSomething()
 
         $this->assertTrue($jq->run('.Foo.Bar == 33'));
     }
+
+    public function testFixProcessBuilderPileup()
+    {
+        $data = json_encode(['Foo' => ['Bar' => 33]]);
+
+        $process1 = $this->getMockBuilder('Symfony\Component\Process\Process')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $process1->expects($this->once())->method('run');
+        $process1->expects($this->once())->method('getOutput')->will($this->returnValue(33));
+
+        $process2 = $this->getMockBuilder('Symfony\Component\Process\Process')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $process2->expects($this->once())->method('run');
+        $process2->expects($this->once())->method('getOutput')->will($this->returnValue(33));
+
+        $pb = $this->getMockBuilder('Symfony\Component\Process\ProcessBuilder')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $pb->expects($this->any())->method('setPrefix')->will($this->returnSelf());
+        $pb->expects($this->any())->method('setArguments')->will($this->returnSelf());
+        $pb->expects($this->any())->method('getProcess')->will($this->onConsecutiveCalls($process1, $process2));
+
+        $dtm = $this->getMockBuilder('JsonQueryWrapper\DataTypeMapper')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $dtm->expects($this->any())->method('map')->will($this->onConsecutiveCalls(33, 33));
+
+        $provider = $this->getMockBuilder('JsonQueryWrapper\DataProvider\Text')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $jq = new JsonQuery($pb, $dtm);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid variables with short names like $jq. Configured minimum length is 3.](https://www.codacy.com/app/estahn/json-query-wrapper/file/1949885713/issues/source?bid=3105591&fileBranchId=3107888#l109)